prestodb/hdp2.6-hive/Dockerfile

```diff
@@ -36,7 +36,15 @@ RUN yum install -y \
 
 # Cleanup
   && yum -y clean all && rm -rf /tmp/* /var/tmp/* \
-  && ln -s /usr/share/java/mysql-connector-java.jar /usr/hdp/current/hive-metastore/lib/mysql-connector-java.jar
+  && ln -s /usr/share/java/mysql-connector-java.jar /usr/hdp/current/hive-metastore/lib/mysql-connector-java.jar \
+
+# Remove extra not needed packages
+  && rpm -e --nodeps $(rpm -qa | grep '^pig') \
+  && rpm -e --nodeps $(rpm -qa | grep '^tez') \
+  && rpm -e --nodeps $(rpm -qa | grep '^zookeeper') \
+  && rpm -e --nodeps $(rpm -qa | grep '^ranger') \
+  && rpm -e --nodeps $(rpm -qa | grep '^atlas') \
+  && rpm -e --nodeps $(rpm -qa | grep '^spark')
```

Comment:
Did you verify none of these is actually needed? E.g. do the tests pass?

Reply:
Nope... there are some tests included here, which check if it is possible to run MR job or hive query.

Reply:
The next step is to release the snapshot version and run travis job (maybe jenkins also).

Reply:
but you will release that snapshot from your branch, and test it prior to merging this in, right?

Reply:
not sure if @losipiuk will not be needing zookeeper for Kafka, please coordinate with him.

Reply:
>but you will release that snapshot from your branch, and test it prior to merging this in, right?

Exactly!

Reply:
> not sure if @losipiuk will not be needing zookeeper for Kafka, please coordinate with him.

IDK. So far the Kafka product tests setup separate container with just single node Kafka. This is similar approach which we are doing for cassandra, mysql and others. It could give as some more coverage if we tested different versions of Kafka coming from different Hadoop distros. But it would be painful. I would skip that presonally until we see huge interest in Kafka connector and possibly see some distro related issues with it.